src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-console.log('hello mate academy!');
+/**
+ *  My first HTML sorted table
+ *
+ * @param {HTMLTableElement} table The table to sort
+ * @param {number} column The index of the column to sort
+ * @param {boolean} asc Determines if the sorting will be in ascending
+ */
+
+function sortTableByColumn(table, column, asc = true) {
+  const dirModifier = asc ? 1 : -1;
+  const tBody = table.querySelector('tbody');
+  const rows = [...tBody.children];
+
+  // Sort each row
+  const sortedRows = rows.sort((a, b) => {
+    const aColText = a.cells[column].innerText;
+    const bColText = b.cells[column].innerText;
+
+    return aColText > bColText ? (1 * dirModifier) : (-1 * dirModifier);
+  });
+
+  // Re-add the newly sorted rows
+  tBody.append(...sortedRows);
+
+  // Remember how the column is currently sorted
+  table.querySelectorAll('th')
+    .forEach(th => th.classList
+      .remove('th-sort-asc', 'th-sort-desc'));
```

Comment:
код просто обрывается, не переноси так
```suggestion
    .forEach(th => (
      th.classList.remove('th-sort-asc', 'th-sort-desc')
    ));
```

Reply:
done